### PR TITLE
Improve log in cmd service dev

### PR DESCRIPTION
**Pull request by NicolasMahe**
In command service dev:
- Revert logrus in favor of log
- Fix missing spaces (fix https://github.com/mesg-foundation/core/issues/403)

### CODE REVIEW

File: cmd/service/dev.go [hunk 1]
```diff
@@ -95,9 +97,21 @@ func listenResults(serviceID string, result string, output string) {
...
-			logrus.Info(aurora.Red(err))
+			log.Println(aurora.Red(err))
 			return
 		}
-		logrus.Info("Receive result", aurora.Green(result.TaskKey), aurora.Cyan(result.OutputKey), "with data", aurora.Bold(result.OutputData))
+		log.Println("Receive result", aurora.Green(result.TaskKey), aurora.Cyan(result.OutputKey), "with data", formatJSON(result.OutputData))
+	}
+}
+
+func formatJSON(data string) string {
```
Comment (krhubert) [inline cmd/service/dev.go]: Move to utils?

Missing docs.

s/formatJSON/colorJSON/ ?

**Comment by krhubert** [comment]
Pls use logrus. Let's keep only one logging package in the app.

**Comment by NicolasMahe** [comment]
We will continue to use logrus for log in the core/daemon.

I switch back to log in the command `service.dev` for UI improvements only.
All the commands are using fmt, not logrus.

File: cmd/service/dev.go (hunk 1 above)
Comment (NicolasMahe) [reply cmd/service/dev.go]: In which utils your suggest to move it? `service/utils`? `cmd/utils`?

Comment (krhubert) [reply cmd/service/dev.go]: cmd/utils

**Comment by NicolasMahe** [comment]
Modification done, @mesg-foundation/core please review

**Pull request merged**